smalld/smalld.py

```diff
@@ -95,17 +95,19 @@ def __init__(
 
     def __getattr__(self, name):
         if name.startswith("on_"):
-            return lambda: self.on_dispatch(t=name.strip("on_").upper())
+            return self.on_dispatch(t=name.strip("on_").upper())
```

Comment:
Won't we need to pass through *args and/or *kwargs here?

Reply:
For what purpose? :thinking: